examples/cryptorealtime/src/main/java/source/CryptoMarketTradeUnboundedReader.java

```diff
@@ -0,0 +1,254 @@
+/**
+ * Copyright 2019 Google
+ * <p/>
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This is not the correct copyright header. If you wrote this code from scratch, it should not mention ASF at all here, it should just be the standard short-form license header, e.g., use the output of `autogen foo.java` here.

Reply:
done in https://github.com/GoogleCloudPlatform/professional-services/pull/191/commits/05ab1977716987c886d5bb7bb221ea4c72cdc1ee